crypto/property/property_local.h

```diff
@@ -13,6 +13,27 @@
 
 typedef int OSSL_PROPERTY_IDX;
 
+typedef enum {
+    OSSL_PROPERTY_OPER_EQ, OSSL_PROPERTY_OPER_NE, OSSL_PROPERTY_OVERRIDE
+} OSSL_PROPERTY_OPER;
+
+struct ossl_property_definition_st {
+    OSSL_PROPERTY_IDX name_idx;
+    OSSL_PROPERTY_TYPE type;
+    OSSL_PROPERTY_OPER oper;
+    unsigned int optional : 1;
+    union {
+        int64_t             int_val;     /* Signed integer */
+        OSSL_PROPERTY_IDX   str_val;     /* String */
+    } v;
+};
+
+struct ossl_property_list_st {
+    int n;
```

Comment:
What is n? use a more descriptive name than this.

Reply:
These were moved verbatim from `property_parse.c`.  I'm *not* ready to change this variable in this PR

Reply:
I wrote this, @levitte just copied it here.

Reply:
*Different PR*

I insist

Reply:
I already marked it as resolved - forgotten already.

Reply:
Addressed this in #15614 too.